src/nvim/fileio.c

```diff
@@ -6979,12 +7010,13 @@ int au_exists(char_u *arg)
 ///
 /// Used for autocommands and 'wildignore'.
 ///
-/// @param pattern    the pattern to match with
-/// @param prog       the pre-compiled regprog or NULL
-/// @param fname      the full path of the file name
-/// @param sfname     the short file name or NULL
-/// @param tail       the tail of the path
-/// @param allow_dirs allow matching with dir
+/// @param pattern pattern to match with
+/// @param prog pre-compiled regprog or NULL
+/// @param fname full path of the file name
+/// @param sfname short file name or NULL
+/// @param tail tail of the path
+/// @param allow_dirs Allow matching with dir
```

Comment:
I though that alignment means more lines changes when changing this place so I tend to avoid too much alignment but in this case I agree that I was mistaken since it won't change much.